### DISCUSSION TITLE
A way to not minify html output?

### DISCUSSION THREAD

**Discussion by stratboy** [vercel/next.js#21293] [Help] [answered]
Hi, I was wondering: is there a way/option to temporarily avoid getting minified html? So that when you view source (instead of console) you'll see pretty indented html?

**Comment by jamesmosier** [comment, marked as answer]
There is currently no option for this. When Next renders the HTML via React's `renderToString` it outputs it to one "minified" line because in HTML whitespace has meaning. Therefore if any unintended spaces are added it would effect the output of the HTML when rendering.
